rpcclient/swap.go

```diff
@@ -41,25 +31,11 @@ func (c *Client) GetOngoingSwap(id string) (*rpc.GetOngoingResponse, error) {
 		OfferID: id,
 	}
 
-	params, err := json.Marshal(req)
-	if err != nil {
-		return nil, err
-	}
-
-	resp, err := rpctypes.PostRPC(c.endpoint, method, string(params))
-	if err != nil {
-		return nil, err
-	}
-
-	if resp.Error != nil {
```

Comment:
The JSON-RPC wrapper around the real request/response is all handled by the Gorilla library now.